src/js/pages/ArbitrationLicense/components/BuyButton.jsx

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react';
+import {Row, Col, Button} from 'reactstrap';
+import PropTypes from 'prop-types';
+import {withNamespaces} from 'react-i18next';
+
+class LicenseBuy extends Component {
```

Comment:
Same here, we can reuse the License component and just add a prop for the name of the button if needed